frontend/src/components/filters/Filters.js

```diff
@@ -2,604 +2,131 @@ import counterpart from 'counterpart';
-          if (foundMatches.length > 0) {
-            filtersToRemove = groupFilter.filter(
-              (toRemove) => toRemove.filterId !== activeStateFilter.filterId
-            );
-          }
-        }
-      });
-    });
-    return activeStateFilters.filter(
-      (asFilterItem) =>
-        !this.isBlacklisted(asFilterItem.filterId, filtersToRemove)
-    );
-  };
-
-  /**
-   * @method isBlackListed
-   * @summary Check if the filterId is found among the filters to remove array
-   * @param {string filterId
-   * @param {array} filtersToRemove
-   */
-  isBlacklisted = (filterId, filtersToRemove) => {
-    let resultValue = false;
-    filtersToRemove.forEach((filterItem) => {
-      if (filterItem.filterId === filterId) {
-        resultValue = true;
-      }
-    });
-    return resultValue;
-  };
-
-  /**
-   * @method sortFilters
-   * @summary ToDo: Describe the method
-   * @param {array} data
-   */
-  sortFilters = (data) => {
-    return {
-      frequentFilters: this.annotateFilters(
-        data
-          .filter((filter) => filter.frequent)
-          .toIndexedSeq()
-          .toArray()
-      ),
-      notFrequentFilters: this.annotateFilters(
-        data
-          .filter((filter) => !filter.frequent && !filter.static)
-          .toIndexedSeq()
-          .toArray()
-      ),
-      staticFilters: this.annotateFilters(
-        data.filter((filter) => filter.static)
-      )
-        .toIndexedSeq()
-        .toArray(),
-    };
-  };
-
-  /**
-   * @method isFilterValid
-   * @summary ToDo: Describe the method
-   * @param {*} filters
-   */
-  isFilterValid = (filters) => {
-    if (filters.parameters) {
-      return !filters.parameters.filter((item) => item.mandatory && !item.value)
-        .length;
-    }
-
-    return true;
-  };
-
-  /**
-   * @method isFilterActive
-   * @summary ToDo: Describe the method
-   * @param {*} filterId
-   */
-  isFilterActive = (filterId) => {
-    const { activeFilter } = this.state;
-
-    if (activeFilter) {
-      // filters with only defaultValues shouldn't be set to active
-      const active = activeFilter.find(
-        (item) => item.filterId === filterId && !item.defaultVal
-      );
-
-      return typeof active !== 'undefined';
-    }
-
-    return false;
-  };
-
-  /**
-   * @method parseToPatch
-   * @summary ToDo: Describe the method
-   * @param {*} params
-   */
-  parseToPatch = (params) => {
-    return params.reduce((acc, param) => {
-      if (
-        // filters with only defaltValue shouldn't be sent to server
-        !param.defaultValue ||
-        JSON.stringify(param.defaultValue) !== JSON.stringify(param.value)
-      ) {
-        acc.push({
-          ...param,
-          value: param.value === '' ? null : param.value,
-        });
-      }
-
-      return acc;
-    }, []);
-  };
-
-  // SETTING FILTERS  --------------------------------------------------------
   /**
    * @method applyFilters
    * @summary This method should update docList
```

Comment:
Well it doesn't really update the docList, since filters are in the store and DL should react automatically.